logback/logback13/build.gradle

```diff
@@ -0,0 +1,32 @@
+dependencies {
+    if (project.ext.targetJavaVersion >= 11) {
+        testImplementation project(':thrift0.18')
+    } else {
+        testImplementation project(':thrift0.17')
+    }
+    api libs.logback13
+}
+
+def logback12Dir = "${rootProject.projectDir}/logback/logback12"
+
+// Copy common files from logback.
+task generateSources(type: Copy) {
+    from "${logback12Dir}/src/main/java"
+    into "${project.ext.genSrcDir}/main/java"
+    exclude "**/LoggingEventWrapper.java"
+    exclude "**/package-info.java"
```

Comment:
Question: Do we need to exclude this? IIRC, there was no problem if we add `'no_aggregation'` flag to the project. 🤔